client/src/components/Message/MessageInput.tsx

```diff
@@ -54,19 +64,25 @@ const MessageInput: React.FC<IMessageInput> = ({
   onEmojiClick,
   ...props
 }) => {
+  const isMobile = !mql.matches;
   const formRef = useRef<HTMLFormElement>();
   const textareaRef = useRef<HTMLTextAreaElement>();
 
+  const imparativeSubmit = (event: any) => {
```

Comment:
Extracted Submit logic to a function so that i can send messages in mobile with "send button"